src/test/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingGivenAllelesUtilsUnitTest.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.walkers.genotyper;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.test.VariantContextTestUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class GenotypingGivenAllelesUtilsUnitTest {
+
+    @Test
+    public void testComposeGivenAllelesVariantContextFromRod() {
+
+        final SimpleInterval loc = new SimpleInterval("20", 10093568, 10093568);
```

Comment:
These loci don't need to be 8-digit numbers.

Reply:
done